src/analytics/formTracker/formTracker.test.ts

```diff
@@ -77,4 +77,14 @@ describe("FormTracker", () => {
     document.body.appendChild(label);
     expect(instance.getFieldLabel()).toBe("test label");
   });
+
+  test("getSubmitUrl should return submit url", () => {
+    const instance = new FormTracker();
+    const form = document.createElement("form");
+    form.action = "/test-url";
```

Comment:
Wondering if you think we should add a test for the edit query? Because it won't be "/test-url" in every case?

Reply:
Hummmm you ask a good question. Let me test something.

Reply:
I added this case in the unit test.

Reply:
Ace, thanks LGTM!